src/argilla/client/feedback/dataset.py

```diff
@@ -505,7 +505,7 @@ def push_to_argilla(self, name: Optional[str] = None, workspace: Optional[Union[
                     datasets_api_v1.add_records(
                         client=httpx_client,
                         id=self.argilla_id,
-                        records=[record.dict() for record in self.__new_records[i : i + PUSHING_BATCH_SIZE]],
+                        records=[record.dict() for record in tqdm(self.__new_records[i : i + PUSHING_BATCH_SIZE])],
```

Comment:
Maybe a nice addition here is to add `desc`, something like

```suggestion
                        records=[record.dict() for record in tqdm(self.__new_records[i : i + PUSHING_BATCH_SIZE], desc="Pushing records to Argilla...")],
```

Reply:
Sure! Noted and added in the next commit.